ext_emconf.php

```diff
@@ -26,12 +26,12 @@
             'crawler' => '9.0.0-9.9.99'
         ],
     ],
-    'state' => 'alpha',
+    'state' => 'beta',
```

Comment:
In would say: stable. It is a version 1.0.0.

Reply:
Changed to stable